static/styles/zulip.scss

```diff
@@ -1088,6 +1088,8 @@ td.pointer {
 }
 
 .selected_message {
+    outline: 0;
```

Comment:
I guess the purpose here is to avoid the duplicate outline?

Reply:
done